submissions/damisiMahmood/html-forms/contactPage.html

```diff
@@ -13,7 +13,8 @@ <h1> I-DAMSELS ELITE SHOPPING </h1>
                 <li><a href="#"> HOME</a></li> 
                 <li><a href="#">ABOUT</a></li> 
                 <li><a href="#">CATGORIES</a></li> 
-                <li><a href="#">BRANDS</a></li> 
+                <li><a href="../html/productPage.html">PRODUCTS</a></li>
```

Comment:
Your html folder is in the capital letters (HTML) but here you are smaller letter (html). Are you sure this PR you are pushing for review works properly?

Reply:
Same goes for other places where you make the same mistake

Reply:
The PR works properly on different browsers, however I have changed the name of the folder to small letters